packages/commerce-sdk-react/src/hooks/ShopperBaskets/query.ts

```diff
@@ -92,6 +94,7 @@ export const usePaymentMethodsForBasket = (
 
     // For some reason, if we don't explicitly set these generic parameters, the inferred type for
     // `Data` sometimes, but not always, includes `Response`, which is incorrect. I don't know why.
+    // @ts-ignore TODO: Fix react query result error generics
```

Comment:
What is the reason we have having this issue? Is you plan to ignore it or will this be fixed in this PR. Or will it be fixed in the refactor ticket you mentioned in a previous comment.